src/containers/Surah/index.js

```diff
@@ -340,18 +340,20 @@ export default class Surah extends Component {
   }
 
   renderLines() {
-    const { lines, options } = this.props;
+    const { lines, options, ayahs } = this.props;
     const keys = Object.keys(lines);
+    const ayahsKeys = Object.keys(ayahs);
 
     return keys.map((lineNum, index) => {
       const nextNum = keys[index + 1];
       const pageNum = lineNum.split('-')[0];
       const line = lines[lineNum];
+      const lastAyahInfo = ayahs[ayahsKeys[ayahsKeys.length - 1]]; // last ayah
```

Comment:
Haha that's not good. I can replicate the bug locally, I assumed each block of ayahs will have their own PageBreak. I will investigate an alternative approach.